docs/source/migration.mdx

```diff
@@ -640,14 +696,21 @@ Note that `startStandaloneServer` does set up body parsing and CORS functionalit
 
 Apollo Server 3 depends on the [`graphql-tag`](https://www.npmjs.com/package/graphql-tag) npm package and re-exports its `gql` template literal tag. The `gql` tag is essentially a caching wrapper around `graphql-js`'s parser, and most IDEs know to treat the contents of `gql` strings as GraphQL.
 
-Apollo Server 4 does not depend on the `graphql-tag` library, nor does it export the `gql` tag. If you want to continue using the `gql` tag, you can directly install `graphql-tag` into your app,  then update your import:
+Apollo Server 4 does not depend on the `graphql-tag` library, nor does it export the `gql` tag. If you want to continue using the `gql` tag, you can directly install `graphql-tag` into your app,  then update your import, replacing this line:
 
 ```ts
-// import { gql } from 'apollo-server' (Removing this line)
-import gql from 'graphql-tag' (Adding this line)
+import { gql } from 'apollo-server';
 ```
 
-However, no Apollo Server APIs (in v3 or v4) actually require you to pass in the parsed document produced by a `gql` template string; for example, the `typeDefs` constructor option allows your schema to be specified either as a string or as a parsed document. Rather than switching to the `graphql-tag` package, you may wish to simply specify your schema as a string (whether inline in your source file, or in a separate `*.graphql` file that you read in your code). Note that most editors that recognize `gql` tags also treat any string starting with `#graphql` as GraphQL.
+with this line:
+
+```ts
+import gql from 'graphql-tag';
+```
+
+(Note that `gql` is a named export of `apollo-server` but is the default export of `graphql-tag`.)
```

Comment:
```suggestion
The `gql` tag is the default export for `graphql-tag`.
```

Reply:
I'm happy to see the wording improved but you're losing the contrast between the two versions here.